### PR TITLE
Story cms build

### PR DESCRIPTION
**Pull request by mattap86**
exported correct db

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    <a class="anchor" id="portfolio"></a>
+    <div class="portfolio">
+        <div class="portfolioText">
+            <h1>PROJECT VAULT</h1>
+            <p>A Peek At My Projects</p>
+            <div class="spacer"></div>
+        </div>
+        <div class="projectBlocks">
+            <?php
+                $db = dbconn();
```
Comment (mporam) [inline index.php:79]: you already have a db connection from edit.php, this is creating two connections

File: php/admin.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    </form>
+    <form action="hide.php" method="post">
+        <h1>HIDE PROJECT</h1>
+        <label>Project Name :</label>
+       <div>
+            <select name="projectId">
+                <option>PLEASE CHOOSE YOUR PROJECT:-</option>
+                <option value="1">pilotShop</option>
+                <option value="2">projectTwo</option>
+                <option value="3">projectThree</option>
```
Comment (mporam) [inline php/admin.php:61]: these should be populated from the DB, otherwise you can only ever delete these and no new ones

File: php/cmsLogin.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?php
+session_start();
```
Comment (mporam) [inline php/cmsLogin.php]: this is part of story 2, why is it in this branch? please delete

File: php/cmsLogin.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    <input type="text" name="name" placeholder="enter your username here" />
+    <input type="text" name="password" placeholder="enter password"/>
+    <input type="submit" />
+</form>
+</body>
+</html>
+
+<?php
+
+if($_SESSION['loggedIn'] == TRUE) {
```
Comment (mporam) [inline php/cmsLogin.php]: PSR2

File: php/functions.php [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+/*
+ * This function selects the 'interests' field from the about_me table retrieved in the getAboutMe function
+ *
+ * @param array $result represents an assoc. array containing the string assigned the key: 'interests'
+ *
+ * @return string $result returns the string assigned the key: 'interests'
+ * @return string returns the string 'error' if the array represented in the variable passed into
+ * the function does not containing a string assigned the key: 'interests'
+ */
+function selectInterestsFromResults(array $result) : string {
```
Comment (mporam) [inline php/functions.php:62]: this function is identical to the one above bar the key, make the key a parameter and you can reuse the same function

File: php/functions.php [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+/*
+ * This function selects the 'qualifications' field from the about_me table retrieved in the getAboutMe function
+ *
+ * @param array $result represents an assoc. array containing the string assigned the key: 'qualifications'
+ *
+ * @return string $result returns the string assigned the key: 'qualifications'
+ * @return string returns the string 'error' if the array represented in the variable passed into
+ * the function does not containing a string assigned the key: 'qualifications'
+ */
+function selectQualificationsFromResults(array $result) : string {
```
Comment (mporam) [inline php/functions.php:79]: dup function

File: php/functions.php [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+ *
+ * @param array $projectResults represents all 'projects' held in the 'project' table of the 'portfolio' database
+ *
+ * @return array $result represents an individual project that is displayed dynamically through the foreach command
+ */
+function displayProjects(array $projectResults) : string
+{
+    $result = '';
+    foreach ($projectResults as $projectResult) {
+        if (array_key_exists('url', $projectResult) && array_key_exists('image_url', $projectResult) && array_key_exists('summary', $projectResult)) {
```
Comment (mporam) [inline php/functions.php]: PSR2 says large if statements should be laid out like:
```
if (
    condition1 &&
    condition2 
) {
    // code
}
```

File: php/functions.php [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+/*
+ * This function uploads all data from the 'projects' table of the 'portfolio' database to index.php in the
+ * projects section of the page
+ *
+ * @param array $projectResults represents all 'projects' held in the 'project' table of the 'portfolio' database
+ *
+ * @return array $result represents an individual project that is displayed dynamically through the foreach command
+ */
+function displayProjects(array $projectResults) : string
+{
```
Comment (mporam) [inline php/functions.php]: PSR2 says put bracket on line above

File: php/functions.php [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+                <a href=' . $projectResult['url'] . '><img class="projectPics" src=' . $projectResult['image_url'] . '></a>
+            </div>
+            <div class="projectSummary projectBox">
+                <h5>' . $projectResult['summary'] . '</h5>
+            </div>
+        </div>';
+        } else {
+            return 'error';
+        }
+    }
```
Comment (mporam) [inline php/functions.php:161]: indentation weird?

File: php/hide.php [hunk 10]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+require_once 'dbFunction.php';
+require_once 'functions.php';
+
+$db = dbconn();
+
+if($_POST['projectId']){
```
Comment (mporam) [inline php/hide.php]: PSR2

File: php/loggedIn.php [hunk 11]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (mporam) [inline php/loggedIn.php]: wrong branch

File: php/loggedIn.php [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+session_start();
+
+$expectedPassword = 'pass';
```
Comment (mporam) [inline php/loggedIn.php]: no password encryption? not very secure.... definutly going to hack you

**Review by mporam — CHANGES_REQUESTED** [review]

File: php/admin.php (hunk 2 above)
Comment (mattap86) [reply php/admin.php:61]: I know but was going for mvp which I felt this covered.

File: php/functions.php (hunk 5 above)
Comment (mattap86) [reply php/functions.php:62]: was okayed as a method by charlie

File: php/hide.php [hunk 13]
```diff
@@ -5,7 +5,7 @@
 
 $db = dbconn();
 
-if($_POST['projectId']){
+if($_POST['projectId']) {
```
Comment (mporam) [inline php/hide.php]: space between `if` and `{`

**Review by mporam — APPROVED** [review]

**Pull request merged**